package.json

```
@@ -12,9 +12,9 @@
   "license": "MPL-2.0",
   "scripts": {
     "analyze:clean": "rimraf stats.json",
-    "analyze": "BABEL_ENV=production NODE_ENV=production node ./internals/scripts/analyze.js",
+    "analyze": "NODE_ENV=production node ./internals/scripts/analyze.js",
```

```suggestion
    "analyze": "ANALZYE=true npm run build",
```
Would make more sense now that we have the BundleAnalyzerPlugin directly in the webpack base config. Then the `analyze:clean` line as wel as `analyze.js` can also be removed.